remarkable/remarkable.go

```diff
@@ -32,10 +32,11 @@ func New() (Remarkable, error) {
 	log.InitLog()
 
 	for i := 0; i < rmAuthRetries; i++ {
-		rm = api.CreateApiCtx(api.AuthHttpCtx())
+		var err error
+		rm, err = api.CreateApiCtx(api.AuthHttpCtx(false, false))
```

Comment:
The AuthHttpCtx here added two arguments:
- the first being `reAuth` which forces a reauth regardless of an existing token. I've gone for false here because that seems  unnecessary?
- the second being `nonInteractive` which would result in failure if there isn't a valid authentication token already. It might be useful to include a flag for running on servers here but I've avoided doing it for now.